docs/initiatives/consent_manager/index.md

```diff
@@ -0,0 +1,36 @@
+---
+sidebar_position: 4
+sidebar_label: Consent Manager
+---
+
+# Consent Manager
+
+Consent Manager is a part of Konnet Stack which handles the entire consent lifecycle. It is responsible for blocking data request transactions and provides authorization based on data owner's consent. 
```

Comment:
Should talk about it's inspiration: https://www.meity.gov.in/sites/upload_files/dit/files/Consolidatep_PPT_02122010.pdf